homework/inner-product/arithmeticAverage.cpp

```diff
@@ -0,0 +1,20 @@
+#include "arithmeticAverage.hpp"
+
+#include <cmath>
+#include <iostream>
+#include <numeric>
+
+double ArithmeticAverage(const std::vector<int>& first, const std::vector<int>& second) {
+    return (static_cast<double>( std::accumulate(first.begin(), first.end(), 0) + std::accumulate(second.begin(), second.end(), 0)) ) / (first.size() + second.size());
```

Comment:
You can also use inner porduct instead of 2 accumulate. Try to think how you can add 2 vectors using inner product :)